### PR TITLE
[TS] LPS-201682 Error Message is copied over between Form Fields

### PR DESCRIPTION
**Pull request by IstvanD**
Dear Team,

Could you please review this pull request?

Thank you and best regards,
István

Related issue: https://liferay.atlassian.net/browse/LPS-201682

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:forms

**Comment by marcelabc** [comment]
ci:stop

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/thunks/fieldChange.es.js [hunk 1]
```diff
@@ -10,6 +10,20 @@ import {disableSubmitButton} from '../utils/submitButtonController.es';
...
 
+const requireUpdatePageFieldNames = [
+	'autocomplete',
+	'inputMask',
+	'numericInputMask',
+	'options',
+	'requireConfirmation',
+	'required',
+	'validation',
+];
```
Comment (alinedoleron) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/thunks/fieldChange.es.js:21]: Hi @IstvanD. According to the previous ci tests I ran on the [initial PR](https://github.com/alinedoleron/liferay-portal/pull/256), some tests have failed. I fixed most of them and I hope I have covered the main ones.

So please, add the following missing fields:

```suggestion
const requireUpdatePageFieldNames = [
	'alphabeticalOrder',
	'autocomplete',
	'dataSourceType',
	'ddmDataProviderInstanceId',
	'expirationDate',
	'hideField',
	'inputMask',
	'limitToOneSubmissionPerUser',
	'multiple',
	'neverExpire',
	'numericInputMask',
	'options',
	'predefinedValue',
	'requireConfirmation',
	'required',
	'sendEmailNotification',
	'validation',
];
```

Sorry for the inconvenience. 
Regards,

**Review by alinedoleron — CHANGES_REQUESTED** [review]
Hi @IstvanD. I have one comment.

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/thunks/fieldChange.es.js (hunk 1 above)
Comment (IstvanD) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/thunks/fieldChange.es.js]: hi @alinedoleron thank you for your feedback. I have added the missing fields.
https://github.com/liferay-forms/liferay-portal/pull/2400/commits/537b54f88435b5bfa91a614e6af4225852a4c730

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:forms

**Comment by alinedoleron** [comment]
ci:test:data-engine

**Review by alinedoleron — APPROVED** [review]
LGTM!

**Comment by alinedoleron** [comment]
> 1. [CanTranslateWebContentWithAllFields](https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/317421//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DELocalization_CanTranslateWebContentWithAllFields_)

**NOT RELATED**

**Comment by alinedoleron** [comment]
ci:test:forms

**Comment by marcelabc** [comment]
Failures not related :+1:

**Comment by marcelabc** [comment]
ci:forward:force

**Comment by marcelabc** [comment]
ci:forward:force

**Pull request closed without merging**